### PR TITLE
WT-11369 Update the wt util compact command with a configuration string

### PR DESCRIPTION
**Pull request by etienneptl**
The file [util_create.c](https://github.com/wiredtiger/wiredtiger/blob/develop/src/utilities/util_create.c) was used as a model for the changes.

### CODE REVIEW

**Review by Sean04 — APPROVED** [review]
LGTM!

File: src/utilities/util_compact.c [hunk 1]
```diff
@@ -15,9 +15,11 @@
 static int
 usage(void)
 {
-    static const char *options[] = {"-?", "show this message", NULL, NULL};
+    static const char *options[] = {"-c config",
+      "a configuration string to be passed to WT_SESSION.compact", "-?", "show this message", NULL,
```
Comment (kommiharibabu) [inline src/utilities/util_compact.c:19]: Can you format the config options for better readability like
 "-c config", "a configuration ...",
 "-?", "Show this message",
 NULL, NULL

**Review by kommiharibabu — APPROVED** [review]
LGTM with a small format change suggestion.

File: src/utilities/util_compact.c (hunk 1 above)
Comment (etienneptl) [reply src/utilities/util_compact.c:19]: @kommiharibabu, I cannot because this is reverted as soon as `s_all` runs.

Comment (kommiharibabu) [reply src/utilities/util_compact.c:19]: ok, thanks for the info.

**Pull request merged**